build/steps/publish-vstest-results.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  testRunTitle: 'TestRun'
+  testResultsFiles: '**/*.trx'
+
+steps:
+- task: PublishTestResults@2
+  displayName: 'Publish ${{ parameters.testRunTitle }} test results'
+  inputs:
+    testRunner: VSTest
+    testRunTitle: ${{ parameters.testRunTitle }}
+    testResultsFiles: ${{ parameters.testResultsFiles }}
+    failTaskOnFailedTests: true
```

Comment:
there isn't really a reason I see to break this out into it's own YAML file just the task directly. See the notes mentioned about just using your parameter to set the enabled property on the task.

Reply:
Thought process behind it was this:
I moved it to its own yaml because then it's common configuration for publishing all trx test run files, so you don't have to update it n-times.
Also if the PublishTestTesults task is updated with additional features some how, you only have to update one file.

Reply:
You have a valid point there... I'll mark this resolved